doc/source/whatsnew/v1.3.0.rst

```diff
@@ -110,49 +110,51 @@ both XPath 1.0 and XSLT 1.0 is available. (:issue:`27554`)
+now allowing custom CSS highlighting instead of the default background coloring (:issue:`40242`).
+Enhancements to other built-in methods include extending :meth:`.Styler.background_gradient`
+to shade elements based on a given gradient map and not be restricted only to
+values in the DataFrame (:issue:`39930`, :issue:`22727`, :issue:`28901`). Additional
 built-in methods such as :meth:`.Styler.highlight_between`, :meth:`.Styler.highlight_quantile`
-and :math:`.Styler.text_gradient` have been added (:issue:`39821`, :issue:`40926`, :issue:`41098`).
+and :meth:`.Styler.text_gradient` have been added (:issue:`39821`, :issue:`40926`, :issue:`41098`).
 
-The :meth:`.Styler.apply` now consistently allows functions with ``ndarray`` output to
-allow more flexible development of UDFs when ``axis`` is ``None`` ``0`` or ``1`` (:issue:`39393`).
+The :meth:`.Styler.apply` method now consistently accepts functions that return an ``ndarray`` to
+allow more flexible development of UDFs when ``axis`` is ``None``, ``0``, or ``1`` (:issue:`39393`).
 
-:meth:`.Styler.set_tooltips` is a new method that allows adding on hover tooltips to
+:meth:`.Styler.set_tooltips` is a new method that allows adding hover tooltips to
 enhance interactive displays (:issue:`35643`). :meth:`.Styler.set_td_classes`, which was recently
 introduced in v1.2.0 (:issue:`36159`) to allow adding specific CSS classes to data cells, has
 been made as performant as :meth:`.Styler.apply` and :meth:`.Styler.applymap` (:issue:`40453`),
 if not more performant in some cases. The overall performance of HTML
 render times has been considerably improved to
-match :meth:`DataFrame.to_html` (:issue:`39952` :issue:`37792` :issue:`40425`).
+match :meth:`DataFrame.to_html` (:issue:`39952`, :issue:`37792`, :issue:`40425`).
 
-The :meth:`.Styler.format` has had upgrades to easily format missing data,
-precision, and perform HTML escaping (:issue:`40437` :issue:`40134`). There have been numerous other bug fixes to
-properly format HTML and eliminate some inconsistencies (:issue:`39942` :issue:`40356` :issue:`39807` :issue:`39889` :issue:`39627`)
+The :meth:`.Styler.format` has been improved to easily format missing data,
+precision, and perform HTML escaping (:issue:`40437`, :issue:`40134`). There have been numerous other bug fixes to
+properly format HTML and eliminate some inconsistencies (:issue:`39942`, :issue:`40356`, :issue:`39807`, :issue:`39889`, :issue:`39627`).
 
-:class:`.Styler` has also been compatible with non-unique index or columns, at least for as many features as are fully compatible, others made only partially compatible (:issue:`41269`).
-One also has greater control of the display through separate sparsification of the index or columns, using the new 'styler' options context (:issue:`41142`).
+Many features of the :class:`.Styler` class are now either partially or fully usable on a DataFrame with a non-unique indexes or columns (:issue:`41269`).
```

Comment:
can use bullet points to improve readability

Reply:
Agreed - while reworking this as bullet points I noticed that much is a duplicate of other lines in the whatsnew. We should either have a single section in Enhancements or single lines in the Other Enhancements (which will be broken into sections, and hence include a Styler section) and Bugfixes.

I lean toward breaking it up (since they aren't all enhancements) into the appropriate sections. cc @attack68 for thoughts.

Reply:
Marking this PR as a draft until this is resolved so it won't be merged.